### ISSUE TITLE
Additional Auto Correct Functionality in JSFormat

### ISSUE THREAD

**Issue by Code785** [sunjw/jstoolnpp#125]
When using the JSON Sort function JSTools will auto correct some format issues such as converting sing quoted keys and string values to double quoted.  Is there any way that functionality can be included in the JSFormat function?  

Additionally would it be possible to auto format common reserved values such as None to null, True to true, and False to false?

![JSTools](https://user-images.githubusercontent.com/105469829/168196058-4edbbbaa-536b-48bf-b658-30385a23be11.PNG)

**Comment by sunjw** [comment]
No, not possible. JSON sort just try its best to parse what you wrote as JSON and re-write it. It's not an auto correct function.
And if you really need this kind of feature, you should check Visual Studio Code or some other IDEs. They can really parse what is in your file, and provide some suggestions.
